### ISSUE TITLE
mint button without using MoralisProvider?

### ISSUE THREAD

**Issue by nidhhoggr** [ethereum-boilerplate/ethereum-boilerplate#187]
So If I want to move forward using this boilerplate to mint NFT's with the current authorized metamask wallet how would I go about doing that? So far all of the documentation I see is using react-moralis MoralisProvider which seems to be incompatible with the way this application is configured.

**Comment by nidhhoggr** [comment]
Most of the minting dapps using Moralis use the MoralisProvider function. 
https://github.com/kjmczk/nextjs-nft-mint-dapp/blob/main/pages/_app.tsx#L8

The problem is that the latest version of Moralis doesnt seem to work with react-moralis which provides the useWeb3ExecuteFunction, or at least is an anti pattern by loading 2 versions of the same library.
https://github.com/kjmczk/nextjs-nft-mint-dapp/blob/main/components/Mint.tsx#L43

Further the most recent version of moralis doesn't provide any contract write functions.
So how I got my minting working using this boilerplate is by using an unwrapped Web3 dependency.
```
import Web3 from 'web3';
const web3 = new Web3(Web3.givenProvider);
const contract = new web3.eth.Contract(nftAbi, contractAddress);
        const response = await contract.methods
          .mint(mintAmount, autoManifest)
          .send({ from: user.address, value: mintPrice });
```

In conclusion, this repo is basically a readonly boilerplate. It would be nice to provide at least one write function in here for completeness. Otherwise consider renaming the repo: ethereum-readonly-boilerplate. 

Closing for now.
